<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head><title>TestNG Report</title><style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}</style></head><body><table><tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr><tr><th colspan="7">Default suite</th></tr><tr><td><a href="#t0">Default test</a></td><td class="num">8</td><td class="num">0</td><td class="num attn">2</td><td class="num">350</td><td></td><td></td></tr></table><table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Default suite</th></tr></tbody><tbody id="t0"><tr><th colspan="4">Default test &#8212; failed</th></tr><tr class="failedeven"><td rowspan="2">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest</td><td><a href="#m0">getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id</a></td><td rowspan="1">1499170631793</td><td rowspan="1">2</td></tr><tr class="failedeven"><td><a href="#m1">getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name</a></td><td rowspan="1">1499170631812</td><td rowspan="1">2</td></tr><tr><th colspan="4">Default test &#8212; passed</th></tr><tr class="passedeven"><td rowspan="8">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest</td><td><a href="#m2">Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active</a></td><td rowspan="1">1499170631629</td><td rowspan="1">100</td></tr><tr class="passedeven"><td><a href="#m3">Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive</a></td><td rowspan="1">1499170631737</td><td rowspan="1">44</td></tr><tr class="passedeven"><td><a href="#m4">getIndividualById_should_return_Individual_object_matching_id</a></td><td rowspan="1">1499170631783</td><td rowspan="1">8</td></tr><tr class="passedeven"><td><a href="#m5">getIndividualByName_should_return_Individual_object_matching_name</a></td><td rowspan="1">1499170631798</td><td rowspan="1">12</td></tr><tr class="passedeven"><td><a href="#m6">getListOfActiveIndividuals_ahould_return_array_list_of_only_active_individual_objects</a></td><td rowspan="1">1499170631818</td><td rowspan="1">3</td></tr><tr class="passedeven"><td><a href="#m7">getListOfInactiveIndividuals_should_return_array_list_of_only_inactive_individual_objects</a></td><td rowspan="1">1499170631823</td><td rowspan="1">4</td></tr><tr class="passedeven"><td><a href="#m8">getListOfIndividuals_should_return_array_list_of_all_individual_objects</a></td><td rowspan="1">1499170631829</td><td rowspan="1">13</td></tr><tr class="passedeven"><td><a href="#m9">getListOfTeams_should_return_a_list_if_Team_object_from_db_json</a></td><td rowspan="1">1499170631848</td><td rowspan="1">34</td></tr></tbody></table><h2>Default test</h2><h3 id="m0">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method TeamsYamlReaderTest.getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest@5034c75a] should have thrown an exception of type class com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m1">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method TeamsYamlReaderTest.getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest@5034c75a] should have thrown an exception of type class com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m2">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m3">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m4">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getIndividualById_should_return_Individual_object_matching_id</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m5">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getIndividualByName_should_return_Individual_object_matching_name</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m6">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getListOfActiveIndividuals_ahould_return_array_list_of_only_active_individual_objects</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m7">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getListOfInactiveIndividuals_should_return_array_list_of_only_inactive_individual_objects</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m8">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getListOfIndividuals_should_return_array_list_of_all_individual_objects</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m9">com.qainfotech.tap.training.resourceio.TeamsYamlReaderTest#getListOfTeams_should_return_a_list_if_Team_object_from_db_json</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p></body></html>